web/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -37,33 +37,63 @@ const activePermitTheme: PermitTheme = {
   headerIconColor: "text-success",
 };
 
+const grayPermitTheme: PermitTheme = {
+  gradient: "gradient-grey",
+  bgColor: "bg-gray-5",
+  textColor: "text-gray-50",
+  borderColor: "border-gray-50",
+  iconColor: "text-gray-50",
+  headerIconColor:"text-gray-50"
+};
+
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
   const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
   const { userData } = useUserData();
 
   useEffect(() => {
     if (props.status === "ACTIVE") {
       setTheme(activePermitTheme);
+      console.log(props.status);
     }
-    if (props.status === "PENDING") {
+    else if (props.status === "PENDING") {
       setTheme(pendingPermitTheme);
+    } else {
+      setTheme(grayPermitTheme);
     }
   }, [props.status]);
 
   const getText = (): string => {
-    if (props.status === "ACTIVE") {
-      return LicenseScreenDefaults.activePermitStatusText;
-    } else if (props.status === "PENDING") {
-      return LicenseScreenDefaults.pendingPermitStatusText;
-    }
-    return "";
-  };
+    return props.status === "ACTIVE" ? LicenseScreenDefaults.activePermitStatusText
```

Comment:
might be nice to have these set up in a lookup table (much like we do those `Lookup` objects for the error texts)